src/steps/5.npmInstall.ts

```diff
@@ -1,4 +1,3 @@
-import { getUserPkgManager } from "../utils/getUserPkgManager"
 import { execa } from "execa"
 
-export default (templateDir: string) => execa(getUserPkgManager(), ["install"], { cwd: templateDir, env: { NODE_ENV: "development" } })
+export default (packageManager: string, templateDir: string) => execa(packageManager, ["install"], { cwd: templateDir, env: { NODE_ENV: "development" } })
```

Comment:
Given you defined the list of package manager, maybe you could declare a litteral union type 

```ts
type PackageManager = 'npm' | 'pnpm' | 'yarn' | 'none';
```

Reply:
That could work actually because there is a type already for PackageManager, I can just add 'none' to it